src/cli/main.rs

```diff
@@ -50,15 +60,14 @@ where
                     }
                     // Run command
                     match cmd {
```

Comment:
Let's keep the same ordering as before. Also, you probably want to bail inside the `run` function rather than here.